packages/common/src/components/Console/Console.stories.tsx

```diff
@@ -2,703 +2,11 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { number } from '@storybook/addon-knobs';
-import Console, { IProps, ConsoleLogSeverities } from './index';
-
-const logs = [
-  {
-    message: 'This is a test of an INFO message.',
-    severity: ConsoleLogSeverities.Info,
-  },
-  {
-    message: 'This is a test of a LOG message.',
-    severity: ConsoleLogSeverities.Log,
-  },
-  {
-    message: 'This is a test of a WARNING message.',
-    severity: ConsoleLogSeverities.Warn,
-  },
-  {
-    message: 'This is a test of an ERROR message.',
-    severity: ConsoleLogSeverities.Error,
-  },
-  {
-    message:
-      "This is a test of an ERROR message. Also, this error message happens to be very very long. Super long. It's only purpose is to be super long. So long that we can test that the log container properly resizes itself and shows all of this super important, meaningful text that will help us understand if this log will be readable by the user.",
-    severity: ConsoleLogSeverities.Error,
-  },
-].map((log, i) => ({ ...log, id: i.toString() }));
-
-const lotsOfLogs = [
```

Comment:
Nico, note how I deleted something like 600 lines of code here... replacing them with just a few lines to generate this array.  I think you were just trying to meet some personal line-count goal that week that you checked this in...

Reply:
more like copying and pasting in non-production code to save time...

Reply:
It's good and all, until you want to start doing more interesting things (e.g,. interjecting other new types of console logs -- would I now need to re-copy-paste back in?).  So I like my approach better.

It also helped me learn that the first line is different in behavior from the second one -- which I think is fascinating!

```
new Array(5).map(a => 100);
[...new Array(5)].map(a => 100);
```